docs/guides/access-groups.mdx

```diff
@@ -217,6 +219,44 @@ Finally, add users to the access group.  You can do this from the access group's
 You can only invite users who are already members of the account.  If you don't see a user on the Add users page, follow the steps in [Invite and manage users](/docs/guides/invite-and-manage-users#invite) to add them to the account first.  After they accept the invitation, you can add them to the access group.
 </Admonition>
 
+## Compare permissions
+
+The following table displays which permissions are granted to account owners and access groups.
+
+| Permissions | Account owner | Qiskit Runtime administrator (access group) | Instance collaborator (access group) |
```

Comment:
```suggestion
| Permissions | Account owner |  IBM Quantum Administrators (access group) | Instance collaborator (access group) |
```

Suggestion to change the name of the group to the one used above in this page.

Reply:
@abbycross I think we need to clarify that "IBM Quantum Administrators" is the access group that you created following the steps above and that "Instance collaborator" is any "collaborators" access group which is automatically created when provisioning/creating an instance using the IBM Quantum Platform.